config/migrations/2024/20240327131000-delete-all-empty-literals.sparql

```diff
@@ -0,0 +1,20 @@
+DELETE {
+  GRAPH ?g { 
+    ?s ?p ?o .
+  }
+}
+WHERE {
+  VALUES ?g {
+    <http://mu.semte.ch/graphs/administrative-unit>
+    <http://mu.semte.ch/graphs/worship-service>
+    <http://mu.semte.ch/graphs/public>
+    <http://mu.semte.ch/graphs/privacy-centric-graph>
+    <http://mu.semte.ch/graphs/worship-privacy-centric-graph>
+  }
+  graph ?g {
+    ?s ?p ?o
+
+    Filter(?o = " " || ?o = "")
```

Comment:
**Question:** Are we sure that a single space and the empty string are the only possibilities? Would it be worth it to filter based on a regex by replacing the filter by (warning untested!!) `FILTER regex(str(?o), "^\\s*$")`.
Note, adding an additional `FILTER (isLiteral(?o))` before the regex filter might be necessary for performance.